nodes/consensus-node.md

```diff
@@ -152,7 +152,7 @@ Here are the summarized recommendations for each node type. There are more detai
 
 ##### Validator node
 
-The reccomendations here are assuming that the validator node is isolated from other responsiblities other than voting and proposing. It is optimized to store as little data as possible. This means that we're retaining the block data for the past ~3 weeks, not indexing transaction, storing the results of the execution of txs, and it's only storing the past two state snapshots. Note that if the validators are connected to a bridge node then the [Bridge Node](#bridge-node) configuration should be used.
+The reccomendations here are assuming that the validator node is isolated from other responsiblities other than voting and proposing. This means that it is not indexing transaction, storing the results of the execution of txs, and it's only storing the past two state snapshots. Note that if the validators are connected to a bridge node then the [Bridge Node](#bridge-node) configuration should be used.
 
 ```toml
```

Comment:
Can we add what file this is for like below?

Reply:
yeah good catch added in 7a43dc82ea08b452a085d415b1f1d98b1d96b7c9